client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -124,7 +124,7 @@ public final class ApiEndpointsV1
 
     public static final String CONNECTED_CLIENT_STATS_ROUTE = API_V1 + CASSANDRA + "/stats/connected-clients";
 
-    public static final String OPERATIONAL_JOBS = "/operational-jobs";
+    private static final String OPERATIONAL_JOBS = "/operational-jobs";
```

Comment:
this is not the route. It is the intermediate string to define routes that share the same prefix. Such intermediate string does not need to be exposed.